### DISCUSSION TITLE
Can't Locate Device ID

### DISCUSSION THREAD

**Discussion by mahmoud-os93** [nVuln/homebridge-lg-thinq#147] [Q&A] [answered]
Hello, might be a silly question but I cannot find the device id : "911fad63-****-****-****-*******" of my Refrigerator and washer.
is it related to S/N or MAC address ?

**Comment by nVuln** [comment, marked as answer]
> Hello, might be a silly question but I cannot find the device id : "911fad63-****-****-****-*******" of my Refrigerator and washer. is it related to S/N or MAC address ?

you can turn on homebridge debug mode, then you can see all device data (device id included)
<img width="453" alt="homebridge debug mode" src="https://user-images.githubusercontent.com/54855446/160263361-73277c3b-6411-491c-9667-53210373956f.png">

**Reply by mahmoud-os93** [reply]
Thank you so much !
